oerconeSup/oercone.proto

```diff
@@ -0,0 +1,30 @@
+Terminator = "\n";
+
+# How Oerlikon Centre-One Pressure Gauge Works:
+## Send a command, then wait 100ms and send an ENQ ("05") to get the reply
+
+
+getPressure {
+	out "PR1\r";
+	wait 100;
+	out "05";
+	in "%d,%f";
+}
+
+getUnits {
+    out "UNI\r";
+	wait 100;
+	out "05";
+    in "%{0|1|2|3}";
+}
+
+setUnits {
+    #UNI,1 = mbar
+    #UNI,2 = Torr
+    #UNI,3 = Pascal
+	#UNI,4 = Micron
+    out "UNI,%{0|1|2|3}\r";
+	wait 100;
+	out "05";
+    in "%{0|1|2|3}";
```

Comment:
You can't write and read from the same record. If you need to do this, either the input or output needs to be [record-redirected](http://epics.web.psi.ch/software/streamdevice/doc/formats.html#redirection).

But usually we would just do the write here and let the read update on it's next scan